statviz/src/components/custom-graphs/BarChartVertical.tsx

```diff
@@ -0,0 +1,177 @@
+import { Grid } from "@visx/grid";
+import { labelProps, tickProps, tooltipStyles } from "../../utils/chart";
+import { Group } from "@visx/group";
+import { scaleLinear } from "@visx/scale";
+import { AxisBottom, AxisLeft } from "@visx/axis";
+import { scaleBand } from "d3-scale";
+import { Bar } from "@visx/shape";
+import { useTooltip, Tooltip } from "@visx/tooltip";
+import { localPoint } from "@visx/event";
+
+type TooltipData = string;
+
+const marginTop = 20;
+const marginLeft = 70;
+const marginRight = 40;
+const marginBottom = 80;
+
+const defaultSettings = {
+  yEndMargin: 0,
+  hideZeroY: true,
+  hideZeroX: false,
+  startWithZero: true,
+};
+
+export interface IBarChartVertical {
+  fields: {
```

Comment:
Why do you use fields? It feels obsolete since it is the only child

Reply:
@MaikNeubert still would like to have an answer here

Reply:
I agree but the BarChartVertical in custom-graphs will be replaced and then removed completely in one of the upcoming PRs with a nivo chart so changing it is not worth it now in my opinion.